### ISSUE TITLE
genie -c is run outside of systemd session

### ISSUE THREAD

**Issue by joyfulmantis** [arkane-systems/genie#13]
Whereas genie -s runs inside a scoped systemd user session (because it's run as a login shell), genie -c runs completely outside of it. This can be verified by running  systemd-cgls in the shell while a genie -c command is still running.

I [patched my fork](https://github.com/joyfulmantis/genie/commit/c80e2705cda4f80c268b0f8926f1b49bbcef6056) to run genie -c commands inside a scoped systemd user session, but it suffers the same issues as the genie -s command, mainly that it loses the working directory. If this solution is considered acceptable I can create a pull request.

**Comment by joyfulmantis** [comment]
I didn't realize that the pull request would track my branch. My bad. I closed the old one, and reopened one with a new branch that just has the sudo fix.

After a bunch of experimenting (and looking at the various pam files) I found that su opens a systemd user session even if it's not running a login shell (thereby still keeping the working directory). I therefore changed both -s and -c to use su and submitted a pull request for it.
